yjit/src/stats.rs

```diff
@@ -193,9 +193,10 @@ make_counters! {
     send_getter_arity,
     send_se_cf_overflow,
     send_se_protected_check_failed,
-    send_splatarray_rhs_too_small,
+    send_splatarray_length_not_equal,
     send_splat_not_array,
-    send_args_splat_variadic,
+    send_args_splat_non_iseq,
+    send_args_splat_cfunc,
```

Comment:
I feel like maybe we should rename these to be more consistent, like have every counter directly related to splat start with `send_splat_`